### DISCUSSION TITLE
SKR Mini E3 V3 SD card Extender

### DISCUSSION THREAD

**Discussion by wazi555** [mriscoc/Ender3V2S1#1137] [Q&A]
So iv taken the plunge and upgraded my standard 422 board with the BTT SKR Mini E3V3, downloaded the precompiled firmware and stuck it on.
Nice features compared to my old Jyers firmware i was running. One issue i have picked up is that this firmware does not seem to support an SD card extender. Anyone else perhaps have the same issue?

**Comment by mriscoc** [comment]
Enable Advanced/Media Update then save settings.

**Reply by wazi555** [reply]
Hi, Thanks for the reply. Unfortunately this option did not work for me.

**Reply by wazi555** [reply]
I still get the warning on the "Home screen" Media Init Fail
This was after enabling the mentioned settings, Saving them and then restarting the printer by using both the "soft" reboot and then after that did not work a Hard reboot. As motioned previously, this is a generic SD adapter and was working with the Jyers firmware. 
I think there may be a bug or different config that needs to be enabled

**Reply by wazi555** [reply]
So in doing research and going through the Marlin code, I have found this - 

 // Enable if SD detect is rendered useless (e.g., by using an SD extender)
  //#define NO_SD_DETECT

Can you confirm if this is enabled in the precompiled version for the Ender3V2-SKRE3V3-BLTUBL-MPC.bin?

**Reply by mriscoc** [reply]
Default configuration has NO_SD_DETECT disabled.

**Reply by GregGFlynn** [reply]
I’m having the same problem, need an extender because I’m upgrading my Sunlu S8 Pro with a SKR Mini E3 V3 and an Ender 3 V2 display so I can use this firmware. Anyone find a cable that works, or another solution?

**Reply by GregGFlynn** [reply]
I found the solution for the extender cable I am using. Turns out there is a 330 ohm resistor on the cable, remove that resistor and the SD card works fine in the extender cable!
![IMG_4148](https://github.com/user-attachments/assets/c07b7bfc-6183-4c85-bea6-c1ea8226d4ed)
![IMG_4150](https://github.com/user-attachments/assets/2a9f01e9-97dc-4a74-8a03-cf60dcac9989)
